packages/contracts-bedrock/test/L1/StandardValidator.t.sol

```diff
@@ -240,6 +279,14 @@ contract StandardValidator_validate_Test is StandardValidator_TestInit {
         assertEq("PROXYA-10,PDDG-DWETH-30,PLDG-DWETH-30", _validate(true));
     }
 
+    /// @notice Tests that the validate function (with an overriden ProxyAdmin owner) successfully returns the right
+    ///         error when the ProxyAdmin owner is not correct.
+    function test_validateOverrideL1PAOMultisig_invalidProxyAdminOwner_succeeds() public {
+        IStandardValidator.ValidationOverrides memory overrides = _defaultValidationOverrides();
+        overrides.l1PAOMultisig = address(0xbad);
+        assertEq("OVERRIDES_L1PAOMULTISIG,PROXYA-10,DF-30,PDDG-DWETH-30,PLDG-DWETH-30", _validate(true, overrides));
```

Comment:
I'd also like to see a test that show only the `OVERRIDES_L1PAOMULTISIG` string returned.